google-http-client/src/main/java/com/google/api/client/http/HttpResponseException.java

```diff
@@ -245,8 +296,11 @@ public Builder setHeaders(HttpHeaders headers) {
     }
 
     /** Returns the HTTP response content or {@code null} for none. */
-    public final String getContent() {
-      return content;
+    public final String getContent() throws IOException {
```

Comment:
This is possibly a breaking change of the API as this builder class is public for some reason. It's fine for binary compatibility but can break source compatibility.